showcase/app/components/shw/flex/index.gts

```diff
@@ -5,6 +5,9 @@
 
 import Component from '@glimmer/component';
 import { htmlSafe } from '@ember/template';
+import { hash } from '@ember/helper';
+import ShwFlexItem from './item';
+import ShwFlexLabel from '../label';
```

Comment:
[chore] Sorry to bother you, maybe I did the wrong suggestion, but this is `ShwLabel` not `ShwFlexLabel`
```suggestion
import ShwLabel from '../label';
```